### PR TITLE
Added dynamic dns guide.

### PR DESCRIPTION
**Pull request by chriscn**
A simple DuckDNS guide that should help new engineers.

### CODE REVIEW

**Comment by 4s3ti** [comment]
will take a look at it, as soon as i can understand why the pipeline failed.

File: docs/guides/dynamic-dns.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Dynamic DNS
+After running PiVPN for a while you may notice that you are unable to connect anymore. This may be due to your Public IP being changed. You then have a dilema, in order to connect to your VPN you need to known the IP; however you can't get that information unless you are on your internal network.  
+
+The solution comes through [Dynamic DNS](https://en.wikipedia.org/wiki/Dynamic_DNS). Which will automatically update an A record on a specific domain name allowing you to always connect.
+
+There are several DynamicDNS providers, some free some not. This tutorial will cover [DuckDNS](https://www.duckdns.org/) however the same principles can apply to other providers. 
+
+## DuckDNS
+1. Head to [DuckDNS](https://www.duckdns.org/) and Sign In with an account. I used GitHub.
```
Comment (4s3ti) [inline docs/guides/dynamic-dns.md:13]: Remove "I used GitHub"

File: docs/guides/dynamic-dns.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title: Dynamic DNS
```
Comment (4s3ti) [inline docs/guides/dynamic-dns.md:1]: Change title to "Dynamic DNS with DuckDNS"

**Review by 4s3ti — COMMENTED** [review]
Generally looks good, i can make some adjustments to the website later, ill merge when you have changed the notes i have added.

**Comment by 4s3ti** [comment]
Also please pull the latest master before any changes so it can fetch the latest changes

**Comment by chriscn** [comment]
Added your changes; let me know anything else you'd like.

**Comment by 4s3ti** [comment]
all seems fine, i will merge later today to a test branch first as i want to make some changes also

**Comment by 4s3ti** [comment]
Merged to test. 

Will make some adjustments on the website itself and release everything

**Comment by 4s3ti** [comment]
Deployed

**Pull request merged**